### PR TITLE
Clarify how to make rollout restarts work with flux-client-side-apply

### PR DESCRIPTION
**Pull request by annaken**

### CODE REVIEW

File: content/en/docs/faq.md [hunk 1]
```diff
@@ -191,12 +191,20 @@ In order for Flux to preserve fields added with kubectl, for example a label or
 you have to specify a field manager named `flux-client-side-apply` e.g.:
 
 ```yaml
-kubectl annotate --field-manager=flux-client-side-apply 
+kubectl --field-manager=flux-client-side-apply label ...
```
Comment (makkes) [inline content/en/docs/faq.md:194]: Why `label` instead of `annotate`?

Comment (annaken) [reply content/en/docs/faq.md:194]: The fact that the rollout restart is failing on an additional annotation made this line super confusing to me - so I just wanted to be clear that this is a generic example and not the instruction about how to edit that restartedAt annotation.

Comment (makkes) [reply content/en/docs/faq.md:194]: I understand, that sounds reasonable. How about we change the sentence right ahead of this example to something like "e.g. for manually adding a label to a resource..."?

**Review by makkes — APPROVED** [review]
Great job, thanks!

**Comment by stefanprodan** [comment]
@annaken can you please [signoff](https://github.com/fluxcd/website/pull/1092/checks?check_run_id=8011591052) your commit and force push it? Thanks

**Pull request merged**